lib/foreman_inventory_upload/async/shell_process.rb

```diff
@@ -4,13 +4,27 @@ module ForemanInventoryUpload
   module Async
     class ShellProcess < ::Actions::EntryAction
       include AsyncHelpers
+      include ::ForemanRhCloud::Async::ExponentialBackoff
 
       def plan(instance_label, more_inputs = {})
         inputs = more_inputs.merge(instance_label: instance_label)
         plan_self(inputs)
       end
 
-      def run
+      def invoke_external_task
+        # Call the polling method from task's framework
+        poll_external_task_with_rescue
+        # supress unexpected task output serialization
+        {}
+      end
+
+      def poll_external_task
+        action
+        # supress unexpected task output serialization
+        {}
+      end
```

Comment:
This gets repeated all over the place. Any reason for being this explicit and not having it inside `::ForemanRhCloud::Async::ExponentialBackoff`?